### PR TITLE
doc: add azure backup configuration to broker configuration page.

### PR DESCRIPTION
**Pull request by rodrigo-lourenco-lopes**
## Description

The broker configuration for azure was missing [here](https://docs.camunda.io/docs/next/self-managed/zeebe-deployment/configuration/broker-config/#zeebebrokerdatabackup).

Changes were added in line to what we have in [here](https://docs.camunda.io/docs/next/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore/#azure-backup-store).

The azure backups are only present in 8.5 minor.

closes https://github.com/camunda/camunda-docs/issues/3485

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
@akeller is OOO, so I'll have a look here 👍

**Comment by christinaausley** [comment]
Is this only applicable to the `next` version of docs? If not, we should likely backport. Let me know if you would like assistance with this.

**Comment by rodrigo-lourenco-lopes** [comment]
> Is this only applicable to the `next` version of docs? If not, we should likely backport. Let me know if you would like assistance with this.

Yes correct, this feature was not backported.

**Review by christinaausley — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

File: docs/self-managed/zeebe-deployment/configuration/broker.md [hunk 1]
```diff
@@ -204,9 +204,9 @@ Use the same configuration on all brokers of this cluster.
...
+| Field | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Example Value |
+| ----- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
```
[the remark this answers is not in the document]
Comment (rodrigo-lourenco-lopes) [reply docs/self-managed/zeebe-deployment/configuration/broker.md:208]: hey @christinaausley  should these suggestions from prettier be committed before merge?

[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/self-managed/zeebe-deployment/configuration/broker.md:208]: Yes 😄

**Review by christinaausley — APPROVED** [review]

**Pull request merged**